script.js

```diff
@@ -21,3 +23,121 @@ window.onload = () => {
+    links: [
+      'https://tufoinnkuo10.github.io/Personal-Portfolio',
+      'https://github.com/tufoinnkuo10/Personal-Portfolio',
+    ],
+  },
+  {
+    Image: './image/dash-data.png',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita',
+    name: 'card 3',
+    technologies: ['Ruby on rails', 'css', 'JavaScript'],
+    links: [
+      'https://tufoinnkuo10.github.io/Personal-Portfolio',
+      'https://github.com/tufoinnkuo10/Personal-Portfolio',
+    ],
+  },
+  {
+    Image: './image/dash-data.png',
+    description:
+      'A daily selection of privatel personalized reads; no accountssign-ups required. Has been the industry standard',
+    name: 'card 4',
+    technologies: ['Ruby on rails', 'css', 'JavaScript'],
+    links: [
+      'https://tufoinnkuo10.github.io/Personal-Portfolio',
+      'https://github.com/tufoinnkuo10/Personal-Portfolio',
+    ],
+  },
+  {
+    Image: './image/websitepro.png',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita',
+    name: 'card 5',
+    technologies: ['Python', 'React', 'bootstrap'],
+    links: [
+      'https://tufoinnkuo10.github.io/Personal-Portfolio',
+      'https://github.com/tufoinnkuo10/Personal-Portfolio',
+    ],
+  },
+  {
+    Image: './image/websitepro.png',
+    description:
+      'A daily selection of privatel personalized reads; no accountssign-ups required. Has been the industry standard',
+    name: 'card 6',
+    technologies: ['Ruby on rails', 'css', 'JavaScript'],
+    links: [
+      'https://tufoinnkuo10.github.io/Personal-Portfolio',
+      'https://github.com/tufoinnkuo10/Personal-Portfolio',
+    ],
+  },
+];
+
+const buttons = document.querySelectorAll('.cproject');
+buttons.forEach((btn, i) => {
+  btn.addEventListener('click', () => {
+    const project = projectContent[i];
+
+    const popup = document.querySelector('.pop-up');
+    popup.style.display = 'flex';
+
+    // get title of popup
+    const popupTitle = popup.querySelector('.title');
+    const popupImage = popup.querySelector('.pop-img');
```

Comment:
* Hi, i can see the popup modal for mobile devices isn't still looking like the original design like the previous code reviewer pointed out kindly adjust the modal on mobile view to look as close as possible as the original design 
* [ORIGINAL]
![image](https://user-images.githubusercontent.com/52497246/131881176-ba010f93-3b26-4fad-b401-8080269d71b6.png)
* [YOURS]
![image](https://user-images.githubusercontent.com/52497246/131881238-c84ddf08-e125-40b7-91d9-81c293c8cd29.png)